### PR TITLE
Bai4

### PR DESCRIPTION
**Pull request by ThanhSonnTL**
## Related Tickets
- [#Task xxxxx: Content](https://edu-redmine.sun-asterisk.vn/issues/xxxx)
## WHAT
- Fragment, Drawable, style
## Evidence (Screenshot or Video)
image
image
image
image
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/blob/master/eng/android/coding_convention_android_kotlin.md | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/sun/android/fragment/SimpleFragment.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import com.sun.android.R
+import com.sun.android.databinding.FragmentSimpleBinding
+
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val yes = 1
+private const val no = 0
+const val radio_button_yes = "radio_button_yes"
+const val radio_button_no = "radio_button_no"
```
Comment (hungld-1302) [inline app/src/main/java/com/sun/android/fragment/SimpleFragment.kt:22]: cái này để làm gì thế e? Sao không cho vào bên trong, với a nghĩ k cần thiết dùng const val trong trường hợp này

File: app/src/main/java/com/sun/android/fragment/SimpleFragment.kt [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import android.widget.TextView
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import com.sun.android.R
+import com.sun.android.databinding.FragmentSimpleBinding
+
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val yes = 1
```
Comment (hungvv-1301) [inline app/src/main/java/com/sun/android/fragment/SimpleFragment.kt:19]: nếu dùng ở 1 file thì để companionobject

File: app/src/main/java/com/sun/android/fragment/SimpleFragment.kt (hunk 1 above)
Comment (hungvv-1301) [inline app/src/main/java/com/sun/android/fragment/SimpleFragment.kt:22]: nếu dùng ở 1 file thì để companionobject

File: app/src/main/res/layout/activity_scorekeeper.xml [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_alignParentTop="true"
+                android:layout_centerHorizontal="true"
+                android:text="@string/team_1"
+                style="@style/ScoreText"/>
+
+            <ImageButton
+                android:id="@+id/decreaseTeam1"
```
Comment (hungld-1302) [inline app/src/main/res/layout/activity_scorekeeper.xml:28]: xem lại convention đặt id nha, Format là Tênview_chứcnăngview
Chỗ này có thể đặt là image_decrease

File: app/src/main/res/layout/activity_scorekeeper.xml [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_alignParentStart="true"
+                android:layout_alignParentLeft="true"
+                android:layout_centerVertical="true"
+                android:onClick="decreaseScore"
+                android:src="@drawable/ic_baseline_remove_24" />
+
+            <TextView
+                android:id="@+id/score_2"
```
Comment (hungld-1302) [inline app/src/main/res/layout/activity_scorekeeper.xml:86]:
```suggestion
                android:id="@+id/text_score"
```

Comment (hungld-1302) [inline app/src/main/res/layout/activity_scorekeeper.xml:86]:
```suggestion
                android:id="@+id/text_score"
```

File: app/src/main/res/layout/activity_scorekeeper.xml [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_centerHorizontal="true"
+                android:layout_centerVertical="true"
+                android:text="@string/initial_count"
+                style="@style/ScoreText"
+                />
+
+            <ImageButton
+                android:id="@+id/increaseTeam2"
```
Comment (hungld-1302) [inline app/src/main/res/layout/activity_scorekeeper.xml:96]: .

File: app/src/main/res/layout/activity_some.xml [hunk 6]
```diff
@@ -33,7 +33,7 @@
         android:hint="@string/textHint">
     </EditText>
     <Button
-        android:id="@+id/btnShareText"
+        android:id="@+id/btnOpenLocaltion"
```
Comment (hungld-1302) [inline app/src/main/res/layout/activity_some.xml:36]:
```suggestion
        android:id="@+id/button_open_location"
```

**Pull request closed without merging**